Package.swift

```diff
@@ -81,6 +81,9 @@ let package = Package(
         .target(name: "CioDataPipelines",
                 dependencies: ["CioInternalCommon", .product(name: "Segment", package: "Segment")],
                 path: "Sources/DataPipeline"),
+        .testTarget(name: "DataPipelineTests",
```

Comment:
Wondering if we name it as `CioDataPipelineTests` to maintain consistency ? Just thinking out loud !

Reply:
None of our test package has `Cio` prefix. It is only for main packages, probably to keep them unique for public apis. So no changes needed here.